src/spyglass/lfp/v1/lfp.py

```diff
@@ -246,3 +206,42 @@ def create_lfp_electrode_group(
                 LFPElectrodeGroup().LFPElectrode.insert1(
                     lfpelectdict, skip_duplicates=True
                 )
+
+
+@schema
+class LFPSelection(dj.Manual):
```

Comment:
Just out of curiosity, is there a reason LFPSelection was moved down here? We typically have the code layout follow the flow of the pipeline for ease of reading.

Reply:
I think this was just because LFPElectrodeGroup needs to be defined before LFPSelection